<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2H7LN3B" ignored="0" name="Test" tests="7" failures="3" timestamp="04 Jan 2020 12:21:34 GMT" time="119.256" errors="0">
  <testcase name="LoginTest" time="3.191" classname="com.inetbanking.testCases.TC_LoginTest_001"/>
  <testcase name="loginDDT" time="13.051" classname="com.inetbanking.testCases.TC_loginDDT_002"/>
  <testcase name="loginDDT" time="5.588" classname="com.inetbanking.testCases.TC_loginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_loginDDT_002.loginDDT(TC_loginDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="5.64" classname="com.inetbanking.testCases.TC_loginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_loginDDT_002.loginDDT(TC_loginDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="12.702" classname="com.inetbanking.testCases.TC_loginDDT_002"/>
  <testcase name="loginDDT" time="5.792" classname="com.inetbanking.testCases.TC_loginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_loginDDT_002.loginDDT(TC_loginDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="AddNewCustomer" time="14.368" classname="com.inetbanking.testCases.TC_AddCustomerTest_003"/>
</testsuite> <!-- Test -->
